components/brave_ads/browser/ads_service_impl.cc

```diff
@@ -716,8 +723,9 @@ void AdsServiceImpl::InitializeNotificationAdsPrefChangeRegistrar() {
       base::BindRepeating(&AdsServiceImpl::NotifyPrefChanged,
                           base::Unretained(this),
                           prefs::kMaximumNotificationAdsPerHour));
-  auto notification_ad_position_callback = base::BindRepeating(
-      &AdsServiceImpl::OnNotificationAdPositionChanged, base::Unretained(this));
+  auto notification_ad_position_callback =
+      base::BindRepeating(&AdsServiceImpl::DoRecordNotificationAdPositionMetric,
```

Comment:
nit: `MaximumNotificationAdsPerHourPrefChanged` to make it future proof for anything that needs to listen to this pref.